recipes/pyvips/meta.yaml

```diff
@@ -0,0 +1,66 @@
+# pyvips supports cffi ABI and API mode
+#
+# During setup.py, it will attempt to compile a generated wrapper against the
+# libvips headers -- this is why it needs pkgconfig in host.
+#
+# At runtime, it will first try to load the compiled wrapper, and if that
+# fails, will fall back to ABI mode, ie. directly opening the libvips shared
+# library and calling into it with cffi.
+#
+# import logging and set the level to debug to see some diagnostics about 
+# this process.
+#
+#       import logging
+#       logging.basicConfig(level = logging.DEBUG)
+
+# Skip win since libvips is not packaged for conda there. It is blocked on 
+# at least gdk-pixbuf.
+
+{% set name = "pyvips" %}
+{% set version = "2.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/libvips/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4dcd1d6689f3faf46b340e70ee238bfd323b9757c5aff8d22a9e05201ebf2cc3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cffi >=1.0
+    - {{ compiler('c') }}
+    - pip
+    - pkgconfig         # [not win]
+    - python
```

Comment:
Only `{{ compiler('c') }}` and `pkgconfig` should go in `build`. The rest belong in `host`.

Reply:
Thanks! I'll try again.